modules/installation-and-upgrade/pages/network-requirements.adoc

```diff
@@ -28,6 +28,46 @@ You also need to ensure that reverse lookups are correctly configured.
 
 For more information about setting up a DNS server, see {sles-base-os-documentation}/cha-dns.html.
 
+== Deployment behind http/https OSI level 7 proxy
+Some environment enforce internet access through a http/https proxy.
+This could be a squid server or similar.
+To allow the {productname} server internet access in such configuration, you need to configure:
```

Comment:
```suggestion
== Deployment behind HTTP or HTTPS OSI level 7 proxy

Some environment enforce internet access through a HTTP or HTTPS proxy.
This could be a Squid server or similar.
To allow the {productname} Server internet access in such configuration, you need to configure the following:
```